datastore/migrate/data/migrations/1707380318_add_columns.down.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE time_series
+	DROP COLUMN parameter_name,
+	DROP COLUMN function,
+	DROP COLUMN period,
+	DROP COLUMN level,
+	DROP CONSTRAINT unique_main,
+	ADD CONSTRAINT unique_main UNIQUE NULLS NOT DISTINCT (version, type, title, summary, keywords,
+		keywords_vocabulary, license, conventions, naming_authority, creator_type, creator_name,
+		creator_email, creator_url, institution, project, source, platform, platform_vocabulary,
+		standard_name, unit, instrument, instrument_vocabulary, link_href, link_rel, link_type,
+		link_hreflang, link_title);
```

Comment:
What does it mean in practice to drop and add a constraint on an existing table. I guess it means rebuilding an index. Not a big deal now, as we are not yet in production, but if you have a big running database, it might be very slow.

Reply:
Do we have a choice, though? There is a way to modify the constraint as such, but not by adding or removing individual columns it seems:

```
ALTER CONSTRAINT constraint_name [ DEFERRABLE | NOT DEFERRABLE ] [ INITIALLY DEFERRED | INITIALLY IMMEDIATE ]
```

Exactly how often will this migration thingy run, though?

Reply:
It's fine for now, but if we make further changes to this while we are running in production, we should test for the downtime.